frontend/src/components/smart/PvPUnderAttackModal.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <div @click="close" class="modalWrapper">
```

Comment:
Why are you using a custom modal and not b-modal (like everywhere else)? Is this needed?

Reply:
Not absolutely, we can write an issue to migrate our custom modals to b-modals if you wish so

Reply:
Please do